### ISSUE TITLE
BBB FPP 8.3.1 Can't Add Stream

### ISSUE THREAD

**Issue by olftboy** [jcrossbdn/fpp-after-hours#31]
I'm running FPP 8.3.  I have installed/uninstalled multiple times.  I've run the manual commands in the ssh window suggested on other issue threads and everything appears to work correctly.  When I click the add new stream button it fades and that is where I'm stuck.  Sure I'm missing something.  Thanks!

**Comment by olftboy** [comment]
Guess I should be more specific.  Like the other post I am on a BBB with 8.3.1 installed.

**Comment by nmiller0113** [comment]
@olftboy good to know that I'm not the only one :) You're experiencing exactly what I am. @jcrossbdn is pretty responsive and I'm absolutely sure he'll figure this out pretty quickly. Considering it's Thanksgiving though, he may not be able to get back to us until after the holiday, but I'm sure we'll hear back soon enough!

**Comment by jcrossbdn** [comment]
Merging the two reports together:

[nmiller0113](https://github.com/nmiller0113) commented [8 hours ago](https://github.com/jcrossbdn/fpp-after-hours/issues/30#issue-2697260354)
I have a fresh install of FPP 8.3.1 and added the After Hours Music Plugin. The plugin install completed successfully as well as the additional install items following the plugin install.

Following the completed install, when I click on +Add Stream, nothing happens. I tried on both Chrome and Safari. I even restarted FPP. No luck. Not sure what else to do, this worked on the previous 7.x install I had running before I ran this brand new install (not upgrade). Thanks!

**Comment by jcrossbdn** [comment]
I have tested on two different pi's (my master which already had the plugin installed and one of my projector pi's which was a fresh install and never had the plugin installed) and both worked.  I am now updating my only BBB to see what happens with it.  This is interesting because the web page should be serving the same content for all hardware platforms and I don't see any warnings popping up in developer mode... I'll dig into it.

**Comment by jcrossbdn** [comment]
I have just installed 8.3.1 on my BBB and still cannot replicate this issue.  It would be helpful to see the issue on either of your devices if you want to do a skype or teams call at some point.  I am Central Standard Time if there is a time that will work I'll throw a meeting link into this thread, just let me know what time is preferred.

**Comment by nmiller0113** [comment]
@jcrossbdn, I'm happy to jump on a Skype or teams call (PST for me). I'm curious, are you running on Windows or Mac...or Linux? I'm on Mac. I'm also accessing mine over wifi...not sure if that would even make a difference, but wanted to clarify all the details. I can try from my daughter's Window machine in a minute too.

**Comment by jcrossbdn** [comment]
I've tried it in Windows in Chrome, Edge and Firefox and on iOS using Safari and it works in all those environments.  Wifi shouldn't make a difference.  I'm not sure if olftboy is Windows or Mac.

**Comment by nmiller0113** [comment]
@jcrossbdn I tested on my daughters windows computer on Chrome and no luck. I also tried on Firefox on my Mac. It's definitely a strange one. Did you want to zoom? I'm in front of my computer now.

**Comment by jcrossbdn** [comment]
>>>zoom meeting was performed<<<

**Comment by jcrossbdn** [comment]
Adding debug image to this thread.
![fpp-after-hours_issue31](https://github.com/user-attachments/assets/947f3301-49b3-4dd4-a5ae-01a9caaff010)

A fix that should correct this error has been pushed.  Plugin will have to be uninstalled using the Plugin Manager and then reinstalled as there was a previous issue with the update functionality that was also just corrected.

**Comment by olftboy** [comment]
Just got home and tried the uninstall/reinstall.  Everything seemed to go smooth, but same issue persists.  Add stream just fades a bit and nothing happens.  I am on Windows and tried Edge/Chrome.  I appreciate your time, but it sure isn't life or death for me to get this running immediately.  Enjoy the holiday and relax! :)

**Comment by jcrossbdn** [comment]
okay thank you we will have to keep investigating.  Nathan is going to allow me remote access to his BBB to try to get this solved in the next few days.

In the meantime, you can use the old user interface which hopefully will work for you by replacing the ending part of the URL from "page=fpp-after-hours.php" to "page=fpp-after-hours-oldui.php"

e.g. on mine the full url is: http://192.168.10.2/plugin.php?plugin=fpp-after-hours&page=fpp-after-hours-oldui.php

* Basically add "-oldui" into the URL

Jason

**Comment by jcrossbdn** [comment]
We were able to solve this issue and have updated the github repository.  Thanks both of you for the reports and thanks again to Nathan for allowing me access to his fpp instance so I could troubleshoot it live on a failing instance.

**Comment by olftboy** [comment]
You are absolutely correct.  Up and streaming here.  Thanks for your tireless support!
